src/main/java/util/CsvAttendanceDataReader.java

```diff
@@ -0,0 +1,58 @@
+package util;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+public class CsvAttendanceDataReader implements AttendanceDataReader {
+    private static final Path FILE_PATH = Paths.get("src", "main", "resources", "attendances.csv");
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+    private static final String DELIMITER = ",";
+    private static final int USERNAME_INDEX = 0;
+    private static final int TIME_INDEX = 1;
+    private static final int LIMIT_INDEX = 2;
+
+    @Override
+    public Map<String, List<LocalDateTime>> loadAttendanceData() {
+        Map<String, List<LocalDateTime>> crewsMap = new HashMap<>();
+        try (Stream<String> lines = Files.lines(FILE_PATH)) {
+            readLines(lines, crewsMap);
+        } catch (FileNotFoundException e) {
+            throw new IllegalStateException("출석 데이터 파일을 찾을 수 없습니다: " + e);
+        } catch (IOException e) {
+            throw new IllegalStateException("출석 데이터 파일을 읽는 중 오류가 발생했습니다: " + e);
+        }
+        return crewsMap;
+    }
+
+    private void readLines(Stream<String> lines, Map<String, List<LocalDateTime>> crewsMap) {
+        Iterator<String> iterator = lines.iterator();
+        if (iterator.hasNext()) {
+            iterator.next();
+        }
+        while (iterator.hasNext()) {
+            String line = iterator.next();
+            processLine(line, crewsMap);
+        }
+    }
```

Comment:
Stream 이니 잘 활용해볼 수 있겠어요~

```suggestion
    private void readLines(Stream<String> lines, Map<String, List<LocalDateTime>> crewsMap) {
        lines.skip1(1)
            .forEach(~~)
    }
```

Reply:
제가 아직 stream 에 익숙하지 않아서 해당 부분을 작성할 때는 아예  stream을 고려조차 못했는데 덕분에 엄청 코드가 간단해진 것 같아요! 감사합니다